typescript/main.ts

```diff
@@ -0,0 +1,4 @@
+export const hello: String = "Hello ";
+export const world: String = "World "
```

Comment:
Nit: semicolons after each line (ideally this would pass our linter tests which are not configured for this package)

Reply:
Ok.